### PR TITLE
KOGITO-1644 Move docker tests to failsafe

### PR DESCRIPTION
**Pull request by jstastny-cz**
Keeping testcontainers, but moving docker dependent tests to integration phase using failsafe plugin.

### CODE REVIEW

**Review by evacchi — APPROVED** [review]

File: process-usertasks-with-security-oidc-quarkus/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Process user task orchestration
+# Kogito user task orchestration
```
Comment (ricardozanini) [inline process-usertasks-with-security-oidc-quarkus/README.md]: Why not `Process`?

**Review by ricardozanini — APPROVED** [review]
One minor comment though

File: process-usertasks-with-security-oidc-quarkus/README.md (hunk 1 above)
Comment (jstastny-cz) [reply process-usertasks-with-security-oidc-quarkus/README.md]: Wrong conflict resolution :/

Comment (jstastny-cz) [inline process-usertasks-with-security-oidc-quarkus/README.md]:
```suggestion
# Process user task orchestration
```

File: process-usertasks-with-security-oidc-springboot/README.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ You will need:
 
 To start a Keycloak Server you can use Docker and just run the following command:
 
-```sh
-docker run -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin  -e KEYCLOAK_IMPORT=/tmp/kogito-realm.json -v <kogito-quickstarts_absolute_path>/kogito-usertasks-with-security-oidc-springboot/config/kogito-realm.json:/tmp/kogito-realm.json -p 8280:8080  jboss/keycloak
+```
```
Comment (jstastny-cz) [inline process-usertasks-with-security-oidc-springboot/README.md]:
```suggestion
```sh
```

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**